### PR TITLE
CIV-4323: How do you want to claim interest?

### PR DESCRIPTION
**Pull request by portizdev**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-4323

### Change description ###
- Screen: 'How do you want to claim interest?'
- Refactor move files under public/claim to /claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -1030,6 +1030,12 @@
       "REACHING_SETTLEMENT": "Setlo’r anghydfod",
```
Comment (commitBlob) [inline src/main/modules/i18n/locales/cy.json:1030]: line numbers don't seem to correspond with en.json

File: src/main/routes/features/claim/interest/interestTypeController.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import express from 'express';
```
Comment (commitBlob) [inline src/main/routes/features/claim/interest/interestTypeController.ts:1]: import {NextFunction, Request, Response, Router} from 'express';

File: src/main/views/features/claim/interest/interest-type.njk [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{% extends "claim-details-tpl.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "../../../macro/contact-us-for-help.njk" import contactUsForHelp %}
+{% from "../../../macro/error-summary.njk" import errorSummary %}
+{% from "../../../macro/page-title.njk" import setPageTitle %}
+{% from "../../../macro/csrf.njk" import csrfProtection %}
+
+{% block pageTitle %}
+  {{ setPageTitle('PAGES.HOW_CLAIM_INTEREST.PAGE_TITLE') }}
```
Comment (commitBlob) [inline src/main/views/features/claim/interest/interest-type.njk:10]: `HOW_CLAIM_INTEREST` not sure how to feel about this naming.
I would opt into `YOUR_INTEREST_OPTIONS` or something more readable

File: src/test/unit/services/features/claim/claimantDetailsService.test.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import * as draftStoreService from '../../../../../main/modules/draft-store/draftStoreService';
+import   * as draftStoreService from '../../../../../main/modules/draft-store/draftStoreService';
```
Comment (commitBlob) [inline src/test/unit/services/features/claim/claimantDetailsService.test.ts:1]: remove this space

File: src/test/unit/views/features/claim/interest-type.test.ts [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+const jsdom = require('jsdom');
+const {JSDOM} = jsdom;
+jest.mock('../../../../../main/modules/oidc');
+jest.mock('../../../../../main/modules/draft-store');
+
+describe('Interest Type View', () => {
+  const citizenRoleToken: string = config.get('citizenRoleToken');
+  const idamUrl: string = config.get('idamUrl');
+
+  beforeEach(() => {
```
Comment (commitBlob) [inline src/test/unit/views/features/claim/interest-type.test.ts:17]: beforeAll

File: src/test/unit/views/features/claim/interest-type.test.ts [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    it('should display save and continue button', () => {
+      const button = mainWrapper.getElementsByClassName('govuk-button')[0];
+      expect(button.innerHTML).toContain('Save and continue');
+    });
+  });
+
+  describe('on POST', () => {
+    let htmlDocument: Document;
+
+    beforeEach(async () => {
```
Comment (commitBlob) [inline src/test/unit/views/features/claim/interest-type.test.ts:65]: beforeAll

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (portizdev) [reply src/main/modules/i18n/locales/cy.json:1032]: Done! Adding some missing translations from other tasks

File: src/main/routes/features/claim/interest/interestTypeController.ts (hunk 2 above)
Comment (portizdev) [reply src/main/routes/features/claim/interest/interestTypeController.ts]: Done

File: src/main/views/features/claim/interest/interest-type.njk (hunk 3 above)
Comment (portizdev) [reply src/main/views/features/claim/interest/interest-type.njk]: Changed to "INTEREST_CLAIM_OPTIONS"

File: src/test/unit/services/features/claim/claimantDetailsService.test.ts (hunk 4 above)
Comment (portizdev) [reply src/test/unit/services/features/claim/claimantDetailsService.test.ts]: Done

File: src/test/unit/views/features/claim/interest-type.test.ts (hunk 5 above)
Comment (portizdev) [reply src/test/unit/views/features/claim/interest-type.test.ts]: Done

File: src/test/unit/views/features/claim/interest-type.test.ts (hunk 6 above)
Comment (portizdev) [reply src/test/unit/views/features/claim/interest-type.test.ts]: Done

**Review by TatsyMaslova — APPROVED** [review]

**Review by commitBlob — APPROVED** [review]

**Pull request merged**